milli/src/lib.rs

```diff
@@ -88,7 +88,7 @@ pub fn absolute_from_relative_position(field_id: FieldId, relative: RelativePosi
 pub fn obkv_to_json(
     displayed_fields: &[FieldId],
     fields_ids_map: &FieldsIdsMap,
-    obkv: obkv::KvReaderU16,
+    obkv: &obkv::KvReaderU16,
```

Comment:
As `KvReaderU16` impl `Copy` I am not sure why you changed this function signature. Could you rollback that change as it breaks the API, please?

Reply:
Sure thing! That's just a by-product of me still learning Rust--not sure when it's best to rely on clone/copy or just pass by reference. I was figuring passing by reference would help save the copy operation during runtime, but that is probably just a micro-optimisation :).